Draft essay

NecroReindeer opened the pull request.

NecroReindeer committed:
Filled in proposal form

NecroReindeer committed:
Essay alpha version. Unedited.

NecroReindeer committed:
Essay beta version

NecroReindeer committed:
Update README.md

Adrir commented on comp110_architecture.tex:
```diff
@@ -0,0 +1,80 @@
+
+\maketitle
+
+\abstract{There are a number of diverse approaches to level generation for 2D platform games, each placing emphasis on a different aspect of level generation. Chunk-based approaches such as Occupancy Regulated Extension tend to favour control and variation, whilst rhythm-based approaches such as that demonstrated by Launchpad favour playability and enjoyability. There are also evolutionary approa ...
```
I would not mention names of other authors (or cite work) in abstracts. I would remove the "as demonstrated by" clause.

Adrir commented on comp110_architecture.tex:
```diff
@@ -0,0 +1,80 @@
+\abstract{There are a number of diverse approaches to level generation for 2D platform games, each placing emphasis on a different aspect of level generation. Chunk-based approaches such as Occupancy Regulated Extension tend to favour control and variation, whilst rhythm-based approaches such as that demonstrated by Launchpad favour playability and enjoyability. There are also evolutionary approa ...
+
+\section{Introduction}
+Procedural content generation for 2D platform games is a relatively new field in comparison to games of other genres~\cite{compton:platform}. Many diverse approaches to this problem have been developed, each having advantages and disadvantages.~\cite{horn:comparative}. There are chunk-based approaches which rely on hand-designed chunks of level; rhythm-based models that generate a level based on  ...
```
Full stop before \cite{horn:comparative} can be removed.

"Level of control" rather than "amount of control".

Adrir commented on comp110_architecture.tex:
```diff
@@ -0,0 +1,80 @@
+
+\section{Comparison}
+
+\subsection{Overview}
```
The overview is clear an format is excellent because it emphasizes analysis over description. I would suggest introducing figures to illustrate the algorithms and how they work as this will improve the context of your comparisons for readers who are unfamiliar with the works you are citing.

Adrir commented on comp110_architecture.tex:
```diff
@@ -0,0 +1,80 @@
+
+
+\section{Conclusion}
+Whilst ORE provides great control over the design of a generated level, Launchpad and Sorenson et al.'s approach provide greater flexibility and reliability with little effort. Sorenson et al.'s approach perhaps provides greater control and flexibility than launchpad; parameters are able to easily be altered and level sections can be designed by hand, in addition to providing control over compone ...
```
Make a stronger more explicit recommendation in the conclusion. A tigher and more restrictive context (which you come up with yourself) may help distinguish one algorithm over the others (e.g., small indie developer vs large AAA studio).

NecroReindeer committed:
Draft for peer review

Draft for peer review. Added figures and made recommendation more
specific. Not yet thoroughly proof-read.

NecroReindeer committed:
Delete launchpad.png

NecroReindeer committed:
Update README.md
